services/blockchain-connector/config.js

```diff
@@ -112,4 +112,11 @@ config.job = {
 	},
 };
 
+config.verifyClientConnection = {
+	// Every n number of blocks, verify if client connection is alive
+	numOfBlocksVerify: process.env.NUM_BLOCKS_VERIFY || 15,
+	// Every n number of blocks, reset client connection cache
+	numOfBlocksReset: process.env.NUM_BLOCKS_RESET || 10,
```

Comment:
Why do we need two of them? Only one should be enough 🤔 
We must reset the connection only when we do not receive events for specified number of blocks (i.e. for configured numBlocks * blockTime).